testsuite/features/step_definitions/common_steps.rb

```diff
@@ -338,7 +337,6 @@
 end
 
 Given(/^I am on the manage software channels page$/) do
-  step %(I am authorized as "testing" with password "testing")
```

Comment:
Does this work? Because visiting the direct link opens a new session that redirects you to the login page right?

Reply:
Yes, this works because opening a new page, keep the cookies of the authenticated user.